### PR TITLE
Earthquake HW

### PR DESCRIPTION
**Pull request by Leah2356**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Earthquakes
+
+Application to display Recent Earthquake data by querying [Earthquake USGS](https://earthquake.usgs.gov/earthquakes/feed/v1.0/geojson.php) using RxJava, Retrofit and GSON.
+
+![gui](/Users/leahfeiner/IdeaProjects/feiner-earthquake-2024/screenshots)
```
Comment (andrewoid) [inline README.md]: This doesn't work

File: src/main/java/feiner/earthquake/EarthquakeFrame.java [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+                        ex.printStackTrace();
+                    }
+                }
+            }
+        });
+    }
+
+    public static void main(String[] args) {
+    new EarthquakeFrame().setVisible(true);
+}
```
Comment (andrewoid) [inline src/main/java/feiner/earthquake/EarthquakeFrame.java]: Feels like your formatting is off here

**Comment by Leah2356** [comment]
I fixed the formatting. For some reason, the screenshots don't work on my
readme, and I've tried many times. I attached it below, thank you!
I can also add it to my comments on github if that's better.

File: README.md (hunk 1 above)
Comment (andrewoid) [reply README.md]: This file path doesn't exist on GitHub. 

And it's pointing to a directory, not an image.

**Comment by Leah2356** [comment]
I fixed the directory, thank you.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Earthquakes
+
+Application to display Recent Earthquake data by querying [Earthquake USGS](https://earthquake.usgs.gov/earthquakes/feed/v1.0/geojson.php) using RxJava, Retrofit and GSON.
+
+![GUI Screenshot](/Users/leahfeiner/IdeaProjects/feiner-earthquake-2024/screenshots/Screen Shot 2024-05-02 at 5.31.39 PM.png)
```
Comment (andrewoid) [inline README.md]: This will only work on your machine.

**Comment by Leah2356** [comment]
I replaced the directory with the file name.

**Review by andrewoid — APPROVED** [review]

**Pull request merged**